app/routes.py

```diff
@@ -1,2 +1,101 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, request
 
+planets_bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
+
+
+@planets_bp.route("", methods=["POST", "GET"])
+def handle_planets():
```

Comment:
Consider making separate functions for the get and post (the GET method is often called `index`, while the POST method is often called `create`). This allows each function to focus on a single piece of functionality, and avoids the need for the if within the route method.